frontend/simple/views/DesignSystem.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div class="section container">
+    <h1 class="title is-1">Style Guide</h1>
+    <div class="columns is-multiline">
+      <div class="column is-4">
+        <h2 class="title is-4 gi-title">Typography</h2>
+        <p class="title is-1">Title is-1</p>
+        <p class="title is-2">Title is-2</p>
+        <p class="title is-3">Title is-3</p>
+        <p class="title is-4">Title is-4</p>
+        <p class="title is-5">Title is-5</p>
+      </div>
+
+      <div class="column is-4">
+        <h2 class="title is-4 gi-title">Text</h2>
+        <p>In an ideal world this website wouldn’t exist, a client would acknowledge the importance of having web copy before the design starts. Needless to say it’s very important, content is king and people are beginning to understand that. </p>
```

Comment:
I don't think this sad copy is relevant here, can we have some happy lorem ipsum on a design system page instead? :)

Reply:
Done :D